package.json

```diff
@@ -56,9 +56,9 @@
     "uncontrollable": "7.2.1"
   },
   "peerDependencies": {
-    "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "prop-types": "^15.8.1",
+    "react": "^18.0.0",
```

Comment:
@PKulkoRaccoonGang To confirm, all of Paragon works when using React 18? 🤔 

Also, as is, this change would cause a peer dependency warning with itself, as it expects React 18, but React 16 is still installed in `devDependencies`.

Most of the Open edX platform is on React 16 in addition to some consumers already using React 17, so we will need to continue to support these versions via a peer dependency range and in the components themselves, e.g.

```
{
  "react": "^16.8.6 || ^17.0.0",
  "react-dom": "^16.8.6 || ^17.0.0"
}
```

Also, `prop-types` as a peer dependency is... odd. I feel like we should probably move it to be in `devDependencies` only most likely.

Reply:
Thanks, done. I checked all components after the new changes. Paragon components work well.